client/fe/webpack/moduleRules.js

```diff
@@ -160,10 +160,11 @@ const moduleRules =
         {
           loader: 'sass-loader',
           options: {
-            implementation: require('sass-embedded'),
-            additionalData: `$url-path-prefix: '${config.urlPathPrefix}';`,
+            api: 'modern-compiler',
             sassOptions: {
-              includePaths: [
+              implementation: require('sass-embedded'),
+              additionalData: `$url-path-prefix: '${config.urlPathPrefix}';`,
```

Comment:
When i tried it like this, the build failed with some error about scss variables inside css files. My suggestion would be to drop this line entirely and hardcode the path in the project that uses the var.

Reply:
seriousely? My build builds.

Reply:
then lets go with it!